README.md

```diff
@@ -1383,6 +1383,7 @@ Table of Contents
 ## Other Free Resources
 
   * [education.github.com](https://education.github.com/pack) — Collection of free services for students. Registration required
+  * [eu.org](https://nic.eu.org) — Free eu.org domain. Request is usually approved in 14 days.
```

Comment:
Correction: It's a subdomain, not a domain

Reply:
no it is a second level domain like co.uk

Reply:
![image](https://user-images.githubusercontent.com/66728880/122873535-83ad9380-d34f-11eb-82d7-db76bdd874a8.png)